lib/tasks/dev.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+  desc 'Generate test data'
+  task :generate_data, [:number] => :environment do |_, args|
+    setargs(args)
+    language_codes = %w[vn en jp lao]
+    programming_language_codes = %w[ruby php java net assembly]
+    @args[:number].times do
+      email = Faker::Internet.email
+      developer = FactoryBot.create(:developer, email: email)
```

Comment:
If a variable is referred only once and defining a variable doesn't improve the readability, you don't need to use a variable.
You can do as follows.
```
FactoryBot.create(:developer, email: Faker::Internet.email)
```